recipes/pytest-mypy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pytest-mypy" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5a5338cecff17f005b181546a13e282761754b481225df37f33d37f86ac5b304
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - mypy >=0.500
```

Comment:
In general, only `pip` and `python` are enough for the `host` section. Is there a reason why the other dependencies are there as well?

Reply:
no, my bad. I just have these because it was auto generated by the conda skeleton command. I will remove them.